src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Value/DateOnlyConverter.cs

```diff
@@ -57,15 +57,23 @@ private static DateOnly ReadCore(ref Utf8JsonReader reader)
 
         public override void Write(Utf8JsonWriter writer, DateOnly value, JsonSerializerOptions options)
         {
+#if NET8_0_OR_GREATER
+            Span<byte> buffer = stackalloc byte[FormatLength];
```

Comment:
Could the new implementation be used in other date/time types as well? There's DateTime/DateTimeOffset support baked into `Utf8JsonWriter` and there's also `TimeSpan`/`TimeOnly` implementations being used in the converter level.

Reply:
It could, but those sites already use Utf8Formatter APIs that now just redirect into the same API we'd be calling. We'd end up ifdefing here for not much benefit.